docker/ubuntu-18.04.dockerfile-todo

```diff
@@ -19,8 +19,10 @@ RUN <<_DEPS
 #!/bin/bash
 apt-get update -y
 apt-get install -y --no-install-recommends \
-  software-properties-common=0.96.24.32.18
-add-apt-repository ppa:ubuntu-toolchain-r/test
+  software-properties-common
+add-apt-repository ppa:ubuntu-toolchain-r/test -y
+add-apt-repository ppa:savoury1/boost-defaults-1.74 -y
```

Comment:
This applies to all dockerfiles... We can't use this repository as it doesn't have arm64 support. That's why we're bootstrapping boost in the 18.04 dockerfile... I'm not sure we want to build boost with qemu though for arm64.

See line 71.